### ISSUE TITLE
Submit to various places

### ISSUE THREAD

**Issue by chadwhitacre** [jashkenas/lil-license#1]
- [ ] [OSI](https://opensource.org/approval)
- [ ] [SPDX](https://spdx.org/spdx-license-list/request-new-license)
- [ ] [GitHub](https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-a-license)
- [ ] [GNU](https://www.gnu.org/licenses/license-list.en.html)

------------------

*was*

> The goal of the OSI License Review Process is to ensure that licenses and software labeled as "open source" conforms to existing community norms and expectations. For that reason, all licenses must go through a public review process described below. The OSI Board is happy to consult with entities in advance to help them navigate the process and improve their license, but formal approval requires going through license-review.

https://opensource.org/approval

**Comment by jashkenas** [comment]
Yes! After getting the initial round of feedback, I'll be submitting it to OSI, SPDX and the Github licensing gems and machinery. I think, in that order.

If you believe that all should have been done before publishing it — that's certainly one way to go. But I thought this order of operations is more in keeping with the open source spirit.

**Comment by jashkenas** [comment]
Just for reference, here are the other links.

SPDX: https://spdx.org/spdx-license-list/request-new-license

Github: https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-a-license

Also, for bonus points, getting it on the GNU list would be nice:
https://www.gnu.org/licenses/license-list.en.html

**Comment by chadwhitacre** [comment]
Sounds good to me. Issue scope expanded! :-)

**Comment by jhabdas** [comment]
In recent conversation with the SPDX legal team they will not accept a license unless it already has traction. And it's going to be relatively hard for a license to gain traction unless it's a part of SPDX.

I for one am not at all concerned with license proliferation. I care about progress. And as such if I can get some reasonable answers to the following question I will more than gladly switch all of my open source licenses over to The Lil License.

https://opensource.stackexchange.com/questions/6155/benefits-of-the-lil-license
